samples/web/content/manual-test/iframe-apprtc/index.html

```diff
@@ -15,6 +15,6 @@
 </head>
 <body>
   AppRTC in an &lt;iframe&gt; element:<br>
-  <iframe width="90%" height="90%" src="http://apprtc.appspot.com"></iframe>
+  <iframe width="90%" height="1200px" src="http://apprtc.appspot.com"></iframe>
```

Comment:
Should probably be 1200, not 1200px (though 1200px will work – try using the Dev Tools to add a height attribute to [simpl.info/iframe](http://simpl.info/iframe)). 

Otherwise LGTM.